tests/baselines/reference/assignmentCompatWithCallSignatures4.errors.txt

```diff
@@ -3,8 +3,6 @@ tests/cases/conformance/types/typeRelationships/assignmentCompatibility/assignme
     Type '(arg2: { foo: number; }) => any' is not assignable to type '(arg2: Base) => Derived'.
       Types of parameters 'arg2' and 'arg2' are incompatible.
         Type '{ foo: number; }' is not assignable to type 'Base'.
-          Types of property 'foo' are incompatible.

```

Comment:
Yup. I've noted that elsewhere in the PR. Mostly I'm trying to see whether this is an appropriate direction to move in. I need to figure out how to maintain this.